client/src/components/restaurant/restaurant-register.js

```diff
@@ -0,0 +1,224 @@
+                <Form.Label>Name *</Form.Label>
+                <Form.Control
+                  autoFocus
+                  name="name"
+                  value={this.state.name}
+                  onChange={this.handleChange}
+                />
+              </Form.Group>
+              <Form.Group controlId="email">
+                <Form.Label>Email *</Form.Label>
+                <Form.Control
+                  autoFocus
+                  name="email"
+                  type="email"
+                  value={this.state.email}
+                  onChange={this.handleChange}
+                />
+              </Form.Group>
+              {' '}
+              <Form.Group controlId="address">
+                <Form.Label>Address *</Form.Label>
+                <Form.Control
+                  autoFocus
+                  name="address"
+                  value={this.state.address}
+                  onChange={this.handleChange}
+                />
+              </Form.Group>
+              {' '}
+              <Form.Group controlId="workingTime">
+                <Form.Label>Working Time *</Form.Label>
+                <br />
+                <div className="time_from">
+                  From:
+                  <Form.Control
+                    autoFocus
+                    type="time"
+                    value={this.state.time_from}
+                    onChange={(event) => { this.setState({ time_from: event.target.value }); }}
+                  />
+                </div>
+                <div className="time_to">
+                  To:
+                  <Form.Control
+                    autoFocus
+                    type="time"
+                    value={this.state.time_to}
+                    onChange={(event) => { this.setState({ time_to: event.target.value }); }}
+                  />
+                </div>
+                <br />
+                <div className="register_count_container">
+                  <Form.Group controlId="countTables">
+                    <Form.Label>Table count:</Form.Label>
+                    <Form.Control
+                      autoFocus
+                      name="count"
+                      type="number"
+                      value={this.state.tables}
+                      onChange={(event) => { this.setState({ tables: event.target.value }); }}
+                    />
+                  </Form.Group>
+                </div>
+              </Form.Group>
+            </div>
+            <div className="register_map_container">
+              {' '}
+            </div>
+          </div>
+          <Form.Group>
+            <Form.Label>Description *</Form.Label>
+            <Form.Control
+              as="textarea"
+              name="description"
+              placeholder="Write description of your restaurant"
+              value={this.state.description}
+              onChange={(event) => { this.setState({ description: event.target.value }); }}
+            />
```

Comment:
Can we use `onChange={this.handleChange}` instead of `onChange={(event) => { this.setState({ description: event.target.value }); }}` ?

Reply:
yeah but on my pc it didn't work so i replaced it with this feature in future i'll fix it